other/restrict_ingress_defaultbackend/restrict_ingress_defaultbackend.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: kyverno.io/v1
+kind: ClusterPolicy
+metadata:
+  name: restrict-ingress-defaultbackend
+  annotations:
+    policies.kyverno.io/title: Restrict Ingress defaultBackend
+    policies.kyverno.io/category: Sample
+    policies.kyverno.io/subject: Ingress
+    policies.kyverno.io/description: >-
+      An Ingress with no rules sends all traffic to a single default backend. The defaultBackend
+      is conventionally a configuration option of the Ingress controller and is not specified in
+      your Ingress resources. If none of the hosts or paths match the HTTP request in the Ingress
+      objects, the traffic is routed to your default backend. In a multi-tenant environment, you
+      want users to use explicit hosts, they should not be able to overwrite the global default backend
+      service.
+spec:
+  validationFailureAction: audit
+  background: true
+  rules:
+  - name: restrict-ingress-defaultbackend
+    match:
+      resources:
```

Comment:
Please place this under an `any` block as Kyverno will be moving away from non-`any` | `all` blocks in `match` or `exclude` clauses.